R/pkg/R/generics.R

```diff
@@ -152,9 +146,9 @@ setGeneric("getNumPartitions", function(x) { standardGeneric("getNumPartitions")
 # @export
 setGeneric("numPartitions", function(x) { standardGeneric("numPartitions") })
 
-# @rdname persist
-# @export
-setGeneric("persist", function(x, newLevel) { standardGeneric("persist") })
+setGeneric("partitionByRDD", function(x, ...) { standardGeneric("partitionByRDD") })
```

Comment:
I see this with the latest from master:
```
* checking for missing documentation entries ... WARNING
Undocumented S4 methods:
  generic 'cache' and siglist 'RDD'
  generic 'collect' and siglist 'RDD'
  generic 'count' and siglist 'RDD'
  generic 'distinct' and siglist 'RDD'
  generic 'first' and siglist 'RDD'
  generic 'join' and siglist 'RDD,RDD'
  generic 'length' and siglist 'RDD'
  generic 'partitionBy' and siglist 'RDD'
  generic 'persist' and siglist 'RDD,character'
  generic 'repartition' and siglist 'RDD'
  generic 'show' and siglist 'RDD'
  generic 'take' and siglist 'RDD,numeric'
  generic 'unpersist' and siglist 'RDD'
```

We have a partitionBy in WindowsSpec.R?
```
setMethod("partitionBy",
          signature(x = "WindowSpec"),
```